### PR TITLE
[ux] (Keyboard shortcuts) Refactor keyboard shortcuts to split out tutorial

### PR DESCRIPTION
**Pull request by AlexJerabek**
This PR refactors the keyboard shortcuts information into two pages: one containing the overview and general information, and another that has a more streamlined tutorial. This change was based on feedback given during the recent docs bash. It does not incorporate other suggestions and contains little net-new content.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[02dda4b](https://github.com/OfficeDev/office-js-docs-pr/commits/02dda4bd171d8f698bc1e26e653a2524b7ebf09d)_: 

:clock10: **Pending**: waiting for processors ([36](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 36 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by o365devx** [comment]
Docs Build status updates of commit _[02dda4b](https://github.com/OfficeDev/office-js-docs-pr/commits/02dda4bd171d8f698bc1e26e653a2524b7ebf09d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/keyboard-shortcuts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Refactor/docs/design/keyboard-shortcuts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/design/keyboard-shortcuts?branch=pr-en-us-2298) |
[docs/excel/configure-your-add-in-to-use-a-shared-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Refactor/docs/excel/configure-your-add-in-to-use-a-shared-runtime.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/configure-your-add-in-to-use-a-shared-runtime?branch=pr-en-us-2298) |
[docs/includes/keyboard-shortcuts-prerequisites.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Refactor/docs/includes/keyboard-shortcuts-prerequisites.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/design/keyboard-shortcuts?branch=pr-en-us-2298) |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Refactor/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/design/keyboard-shortcuts?branch=pr-en-us-2298) |
[docs/tutorials/keyboard-shortcuts-tutorial.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Refactor/docs/tutorials/keyboard-shortcuts-tutorial.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/tutorials/keyboard-shortcuts-tutorial?branch=pr-en-us-2298) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C30%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202010301906046364-2298%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=TZEW1JU94oByFPDSlyDhW7%2Fq0TI62odV5qus9uODBmw%3D&st=2020-10-30T19%3A01%3A44Z&se=2020-11-30T19%3A06%3A44Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/tutorials/keyboard-shortcuts-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+            })
+            .catch(function (error) {
+                return error.code;
+            });
+    });
+    ```
+
+1. Save your changes and rebuild the project.
+
+   ```command line
```
Comment (Rick-Kirkham) [inline docs/tutorials/keyboard-shortcuts-tutorial.md:124]:
```suggestion
   ```command&nbsp;line
```

**Review by Rick-Kirkham — APPROVED** [review]
Approving with a few nits, but you can merge and leave them for me to do do.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[4b21275](https://github.com/OfficeDev/office-js-docs-pr/commits/4b21275673e88a614bf5c7002e492b5db6178ffa)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/keyboard-shortcuts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Refactor/docs/design/keyboard-shortcuts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/design/keyboard-shortcuts?branch=pr-en-us-2298) |
[docs/excel/configure-your-add-in-to-use-a-shared-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Refactor/docs/excel/configure-your-add-in-to-use-a-shared-runtime.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/configure-your-add-in-to-use-a-shared-runtime?branch=pr-en-us-2298) |
[docs/includes/keyboard-shortcuts-prerequisites.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Refactor/docs/includes/keyboard-shortcuts-prerequisites.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/design/keyboard-shortcuts?branch=pr-en-us-2298) |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Refactor/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/design/keyboard-shortcuts?branch=pr-en-us-2298) |
[docs/tutorials/keyboard-shortcuts-tutorial.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Refactor/docs/tutorials/keyboard-shortcuts-tutorial.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/tutorials/keyboard-shortcuts-tutorial?branch=pr-en-us-2298) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202011062006009682-2298%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=HT6sE2ckFyFbaJ3%2BddI%2FWAh48xhD1OJgSef5BQvq%2FoY%3D&st=2020-11-06T20%3A01%3A38Z&se=2020-12-07T20%3A06%3A38Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by AlexJerabek** [comment]
Thanks @Rick-Kirkham. I've committed your suggestions. Merging now.

**Pull request merged**